pages/facts.html

```diff
@@ -1,12 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link href="../styles/facts.css" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/open-dyslexic" rel="stylesheet">
 </head>
-<body>
+
+<body class="container">
+
+  <header>
+    <nav class="btn-container">
+      <div class="box">
+        <a href="/index.html" class="btn"><span>Home</span></a>
+      </div>
+      <div class="box">
+        <a href="#" class="btn"><span>Facts</span></a>
+      </div>
+      <div class="box">
+        <a href="/pages/gallery.html" class="btn"><span>Gallery</span></a>
+      </div>
+    </nav>
+    <h1>Fun facts</h1>
+  </header>
+
+  <main>
+    <section>
+        <h2>What Are Invisible Pets? </h2>
+      <ul>
+        <li class="list-facts">Invisible pets are companions you can't see but can feel! They’re perfect for anyone with a wild imagination who’s ready to adopt a one-of-a-kind, low-maintenance friend. They can be anything: invisible dogs, cats, dragons, or even something entirely unique to the owner.</li>
+        <li class="list-facts">Invisible pets don’t shed, bark, meow, or require a litter box!</li>
+      </ul>
+      <h2>Meet Our Most Popular Breeds </h2>
+      <ul>
+        <li class="list-facts"><span class="text-facts">Phantomo Cats:</span> Known for their silent purrs, they keep you company without knocking things over.</li>
+        <li class="list-facts"><span class="text-facts">Breezy Beagles:</span> Experts at invisible fetch games, and known for sniffing out imaginary treats.</li>
+        <li class="list-facts"><span class="text-facts">Cloud Dragons:</span> They hover near you like a soft breeze and love to "breathe" invisible fire.</li>
+      </ul>
+      <h2>Invisible Pet Care 101</h2>
+      <ul>
+        <li class="list-facts"><span class="text-facts">Feeding:</span> Invisible pets have refined tastes, but fortunately, they love imaginary food! Just pretend to pour it out, and they’ll be thrilled.</li>
+        <li class="list-facts"><span class="text-facts">Exercise:</span> Invisible dogs love long, pretend walks, while invisible cats enjoy imaginary sunbeams. </li>
+        <li class="list-facts"><span class="text-facts">Bathing:</span> Baths are easy—just pretend to wash them, and they’ll be clean in no time!</li>
+      </ul>
+    </section>
```

Comment:
You use semantic elements appropriately in `facts.html`. Putting "Meet Our Most Popular Breeds", "Meet Our Most Popular Breeds", and "Invisible Pet Care 101" in a single `section` tag, which works! I could also see putting each of these parts into their own `section` elements too.

I don't think you need it here and it's not necessarily correct, but I thought I'd call it out to highlight how different people interpret what elements are needed on a page.